File: gcc/testsuite/gcc.dg/Wno-all.c
/* PR 30437: Test negative of -Wall
   Don't change this without changing Wall.c as well.  */
/* { dg-do compile } */
/* { dg-options "-Wall -Wno-all" } */

void foo(int a)
{
  5 * (a == 1) | (a == 2);  /* { dg-bogus "no effect" "no effect" } */
}

File: gcc/testsuite/gfortran.dg/Wno-all.f90
! PR 30437  Test for negative Wall
! { dg-do run }
! { dg-options "-Wall -Wno-all" }
program main
  character (len=40) &
  c
  c = "Hello, &
         world!" ! { dg-bogus "Warning: Missing '&' in continued character constant" }
  if (c.ne.&
                                   "Hello, world!")&
                               STOP 1;end program main

